src/app/work-item/work-item.service.ts

```diff
@@ -154,10 +153,7 @@ export class WorkItemService {
     let url = this.workItemUrl;
     if (id in this.workItemIdIndexMap) {
       let wItem = this.workItems[this.workItemIdIndexMap[id]];
-      if (process.env.ENV != 'inmemory') {
-        this.resolveComments(wItem);
-        this.resolveLinks(wItem);
```

Comment:
I wonder the URLs you are sending back as response are actually resolvable. It was not in case of inmemory so I added this logical check.